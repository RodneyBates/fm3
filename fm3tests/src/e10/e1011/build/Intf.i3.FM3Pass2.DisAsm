 ItkEOF()
 ItkLeftEnd()
 ItkInterfaceLt(L1(16_1),[1,0])
 ItkDeclScopeLt(L1(16_1))
 ItkOpenScopeLt(L1(16_1))
 ItkOpenDeclListLt(L1(16_1),[22,2])
 ItkTypeDeclLt([22,7])
 ItkDeclNo(D1,[22,7])
 ItkTypeDeclType([22,12])
 ItkDefTopExprNo(L2(16_2),[22,14])
 ItkDeclScopeLt(L2(16_2))
 ItkFieldDeclListLt(L0(16_0),[22,21])
 ItkFieldDeclListRt(L0(16_0),[22,21])
 ItkMethodDeclListLt(L0(16_0),[22,21])
 ItkMethodDeclListRt(L0(16_0),[22,21])
 ItkOverrideListLt(L0(16_0),[22,38])
 ItkOverrideListRt(L0(16_0),[22,38])
 ItkDefTopExprNo(L1(16_1),[22,14])
 ItkDeclScopeRt(L2(16_2))
 ItkTypeDeclRt([22,7])
 ItkOpenDeclListRt(L1(16_1),[22,2])
 ItkOpenScopeRt(L1(16_1))
 ItkDeclScopeRt(L1(16_1))
 ItkInterfaceRt(L1(16_1),[1,0])
 ItkRightEnd()
 ItkBOF()
     0
