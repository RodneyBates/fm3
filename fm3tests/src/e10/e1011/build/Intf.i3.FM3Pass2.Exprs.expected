From unit expression map, Expr No 0 NIL

From unit expression map, Expr No 1 16_****************
 ExpStackLink = NIL
 ExpType = NIL
 ExpRefConstVal = NIL
 ExpScalarConstVal = 0
 ExpLoTypeInfoRef = 
 ExpReachedDeclNoSet = {}
 ExpSelfExprNo = 1
 ExpPosition = [22,14]
 ExpOpcode = OBJECT
 ExpDownKind = EkNull
 ExpUpKind = EkType
 ExpKind = EkObjType
 ExpState = EsUnresolved
 ExpIsConst = FALSE
 ExpConstValIsKnown = FALSE
 ExpIsUsable = TRUE
 ExpIsLegalRecursive = TRUE
 ExpIsDesignator = FALSE
 ExpIsWritable = FALSE
 ExpIsPresent = TRUE
 ExpRefTypeIsUntraced = FALSE
 ExpArrayTypeIsOpen = FALSE
 (* From subtype Expr1OpndTyp: *)
 ExpOpnd1 = NIL
 (* From subtype Expr2OpndTyp: *)
 ExpOpnd2 = NIL
 (* From subtype ExprObjTypeTyp: *)
 ExpObjOverrides = NIL
 ExpObjDecls = ScopeNo 2 at 16_**************** [22,14]
 ExpObjDecls = BkAbsent


From unit expression map, Expr No 2 16_****************
 ExpStackLink = ExprNo 1 at 16_**************** [22,14]
 ExpType = NIL
 ExpRefConstVal = NIL
 ExpScalarConstVal = 0
 ExpLoTypeInfoRef = 
 ExpReachedDeclNoSet = {}
 ExpSelfExprNo = 2
 ExpPosition = [22,14]
 ExpOpcode = <RidNull>
 ExpDownKind = EkNull
 ExpUpKind = EkBrand
 ExpKind = EkBrand
 ExpState = EsUnresolved
 ExpIsConst = FALSE
 ExpConstValIsKnown = FALSE
 ExpIsUsable = TRUE
 ExpIsLegalRecursive = TRUE
 ExpIsDesignator = FALSE
 ExpIsWritable = FALSE
 ExpIsPresent = FALSE
 ExpRefTypeIsUntraced = FALSE
 ExpArrayTypeIsOpen = FALSE
 (* From subtype Expr1OpndTyp: *)
 ExpOpnd1 = NIL
 (* From subtype Expr2OpndTyp: *)
 ExpOpnd2 = NIL
 (* From subtype ExprObjTypeTyp: *)
 ExpObjOverrides = NIL
 ExpObjDecls = NIL
 ExpObjDecls = BkAbsent


From unit expression map, Expr No 3 16_****************
 ExpStackLink = ExprNo 2 at 16_**************** [22,14]
 ExpType = NIL
 ExpRefConstVal = NIL
 ExpScalarConstVal = 0
 ExpLoTypeInfoRef = 
 ExpReachedDeclNoSet = {}
 ExpSelfExprNo = 3
 ExpPosition = [22,14]
 ExpOpcode = <RidNull>
 ExpDownKind = EkNull
 ExpUpKind = EkSupertype
 ExpKind = EkSupertype
 ExpState = EsUnresolved
 ExpIsConst = FALSE
 ExpConstValIsKnown = FALSE
 ExpIsUsable = TRUE
 ExpIsLegalRecursive = TRUE
 ExpIsDesignator = FALSE
 ExpIsWritable = FALSE
 ExpIsPresent = FALSE
 ExpRefTypeIsUntraced = FALSE
 ExpArrayTypeIsOpen = FALSE


